package/MDAnalysis/analysis/base.py

```diff
@@ -27,6 +27,8 @@
 
 import numpy as np
 import logging
+from MDAnalysis.lib.log import ProgressMeter
```

Comment:
Your editor should tell you about that. I use `elpy` with emacs but if you prefer IDE's I hear pycharm is good and has a PEP8 checker included.